gym_snake/envs/snakeGameGym.py

```diff
@@ -48,6 +49,11 @@ def __init__(self, fps: int, use_pygame: bool = True):
 			self.win = pygame.display.set_mode((self.width, self.height))
 			self.clock = pygame.time.Clock()
 
+	def make_snake_rand_pos(self):
```

Comment:
I would make this function more general. Really you are just getting a random position on the board right? So maybe just call it `get_random_pos`
```suggestion
	def get_rand_pos(self):
```